### DISCUSSION TITLE
BLE- Bluetooth Low Energy in INET/OMNET++

### DISCUSSION THREAD

**Discussion by wathsala-vishwajith** [omnetpp/omnetpp#1034] [Q&A] [answered]
Can the MaXiM Framework ported in to INET 4.x? 
It says most of the MaXiM framework is now ported to INET. Does that mean it can theoretically work?
[](https://omnetpp.org/download-items/BLE.html)

**Comment by rhornig** [comment, marked as answer]
No, most of the **functionality** in MiXiM was added to INET. The architecture is considerably different, so sadly you won't be able to use the MiXiM based BLE model in INET. That model has to be also migrated/ported to INET.
